lib/controls/ValueSelector.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+class ValueSelector extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {value, options, className, handleOnChange} = this.props;
+    const children = this.props.children;
+
+    if (children) {
+      return (
+        React.cloneElement(children,
```

Comment:
We can pass in the fallback element as a prop. The control can return that of it wants to use the fallback.